openslides/agenda/static/js/agenda/base.js

```diff
@@ -87,6 +87,16 @@ angular.module('OpenSlidesApp.agenda', ['OpenSlidesApp.users'])
                 getAgendaTitle: function () {
                     return this.title;
                 },
+                getTopicText: function () {
```

Comment:
This is a bad hack but I can live with it.

Proposal: Rename this method to `getCSVExportText`. Then every app or plugin may decide whether to provide extra text or not and it is not semi-hard coded to the topic app.